apache-rat-core/pom.xml

```diff
@@ -44,20 +44,17 @@
           <artifactId>apache-rat-plugin</artifactId>
           <configuration>
             <inputExcludes>
-              <!-- remove if RAT-501 is fixed -->
+              <!-- exclude the definition file(s) -->
               <exclude>**/default.xml</exclude>
               <exclude>**/example-configuration.xml</exclude>
+              <!-- exclude test generation files that contain unapproved license text -->
               <exclude>**/ReporterOptionsProvider.java</exclude>
               <exclude>**/GPLLicenseTest.java</exclude>
-              <!-- /remove if RAT-501 is fixed -->
               <!-- Generated by Eclipse, and not distributed, so ignorable. -->
-              <exclude>bin/**/*</exclude>
-              <exclude>.externalToolBuilders/**/*</exclude>
+              <exclude>bin/**/*</exclude>-->
```

Comment:
@Claudenw After rebasing my commit on the branch is lost .....

Reply:
@Claudenw would you be so kind to fix the "-->" at the end of the line and add a comment that this entry can be removed after updating to releases >0.17.